### DISCUSSION TITLE
BlueSky app doesn't appear like other Social Media in iOS Share Sheet

### DISCUSSION THREAD

**Discussion by reflectivist** [bluesky-social/atproto#3283] [Bluesky Client App]
While OAuth is not ready for prime time it would be a lot better for iOS users sharing content if the iOS App configured itself so iOS could add it to the list of content-accepting apps in the iOS Share Sheet.

**Comment by DavidBuchanan314** [comment]
how does this relate to oauth?

**Reply by reflectivist** [reply]
Sorry I screwed up. I do see bluesky in the share sheet now! I think it used to not be there.

But I mean that I tried to get OAuth to share to BlueSky and hit problems I documented elsewhere, and came to think that BlueSky OAuth is not fully working yet, and so then the option to share content most easily is through the share sheet. "So I was trying to say, hey, at least give us that"
